network/http/client.cpp

```diff
@@ -0,0 +1,173 @@
+#include "client.hpp"
+
+#include "coding/base64.hpp"
+
+namespace om::network::http
```

Comment:
Is om:: namespace necessary/helpful?

Reply:
Does it make sense to start following best practices in the whole code, and only when it is necessary? E.g. when we start working on the SDK? Otherwise, we'll have different mixed approaches in the code that will confuse developers.